### PR TITLE
Games model

### PR DESCRIPTION
**Pull request by ruchagade**
created game model

### CODE REVIEW

**Review by Aremusch — CHANGES_REQUESTED** [review]
`validates :name, uniqueness: true` should be `validates :name, uniqueness: { scope: :guild_id }`

File: db/migrate/20240507094603_create_games.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateGames < ActiveRecord::Migration[7.1]
+  def change
+    create_table :games do |t|
+      t.string :name
```
Comment (bdoughertys) [inline db/migrate/20240507094603_create_games.rb]: add null: false

File: app/models/game.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Game < ApplicationRecord
+  belongs_to :guild
+
+  has_many :game_members
```
Comment (Aremusch) [inline app/models/game.rb]: has many :members, through :game_members

**Review by Aremusch — CHANGES_REQUESTED** [review]
`validates :game_icon, presence: true` should be `validates :game_icon, :name presence: true`

**Review by Aremusch — APPROVED** [review]
👍

**Pull request merged**